appveyor.yml

```diff
@@ -32,3 +32,6 @@ deploy:
     skip_symbols: true
     on:
       branch: master
+
+# For PRs, skip the branch run and only run against the virtual PR merge.
+skip_branch_with_pr: true
```

Comment:
If we do this, then we loose the  NuGet packages artifacts. Should we add an upload-artifact step to the ci build?

Reply:
I didn't think about the artifacts. We still get NuGet packages artifacts for the "virtual" PR build. Do you use the packages from the branch build? (it was mostly to avoid having to run both (sequentially) on PRs).

Alternatively, we can try to increase `max_jobs` to 2?

Reply:
I have used the artifacts of PR builds in the past, but probably can just create the package locally as I need to have them in my local NuGet folder anyhow.

Reply:
Note that we will still have artifacts of the PR builds, but now only from the "PR" build - before we had two AppVeyor builds per PR (one for branch build and one for the merge build)
![image](https://github.com/nunit/nunit.analyzers/assets/1007631/a59e5a60-bf6f-4881-a6a5-f4676259e3f7)
So this PR should remove the one build - `continuous-integration/appveyor/branch`

Reply:
I've only seen one, e.g. #642.

Reply:
Odd. I wonder if the branch builds are removed after some time? I can see two on #646, but only one on e.g. #642 or #641 (note that is the missing one that I'm trying to remove)

Reply:
I'll try this out and if we loose the artifacts then I'll revert. Hopefully, this will just skip one of the builds, thus reducing the wait on AppVeyor on PRs.